documentation/disaster-recovery-testing.md

```diff
@@ -27,7 +27,7 @@ This document covers the Disaster Recovery testing procedure for applications ho
 ## Documentation requirements
 
 Copy the template DR testing document which will be a record of the steps run, time taken, and any issues
```

Comment:
Could rephrase as:
```
Copy the [template DR testing document](ttps://educationgovuk.sharepoint.com/:w:/r/sites/TeacherServices/Shared%20Documents/DR%20tests/DR%20test%20template.docx?d=waba054c48ee644e5ab5a66c784fa3b95&csf=1&web=1&e=CRNjv7)) which will be a record of the steps run, time taken, and any issues.
```
and delete the rest

Reply:
updated